docs/core/rid-catalog.md

```diff
@@ -119,7 +119,7 @@ wish to check if new ones are added, please check back here.
     * `linuxmint.17.2-x64`
     * `linuxmint.17.3-x64`
 
-## OS X RIDs
+## macOS RIDs
```

Comment:
This one is probably wrong. These are indeed OS X RIDs.

The rule is:

- Use macOS for the general usage of the operating system.
- Use OS X if you are talking about El Capitan or earlier.

We don't support macOS Sierra yet, so this is El Capitan.